api/base/serializers.py

```diff
@@ -88,7 +105,35 @@ def to_internal_value(self, data):
         return super(JSONAPIListField, self).to_internal_value(data)
 
 
-class JSONAPIHyperlinkedIdentityField(ser.HyperlinkedIdentityField):
+class AuthorizedCharField(ser.CharField):
+    """
+    Passes auth of the logged-in user to the object's method
+    defined as the field source.
+
+    Example:
+        content = AuthorizedCharField(source='return_content')
+    """
+
+    def __init__(self, **kwargs):
+        super(AuthorizedCharField, self).__init__(**kwargs)
```

Comment:
I think this method can be removed, since it only exists to call the parent method (with no intermediate work or change).